Source/NSOutlineView.m

```diff
@@ -194,8 +223,11 @@ - (id) initWithFrame: (NSRect)frame
 
 - (void) dealloc
 {
+  [GSKeyValueBinding unbindAllForObject: self];
```

Comment:
Do we still need this, now that you have the call on NSControl?

Reply:
I don't think so.  I actually thought I had removed it.

Reply:
Removed.